scripts/container_tools/record_rosbag

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Log colors
+CYAN="\033[0;36m"
+MAGENTA="\033[0;35m"
+NO_COLOR="\033[0m"
+
+# Define the output directory
+OUTPUT_DIR="$ROS_WS/rosbags"
+
+# Generate a date and time prefix in the format YYYY_MM_DD-HH_MM_SS
+DATE_PREFIX=$(date "+%Y_%m_%d-%H_%M_%S")
```

Comment:
It saddens me to see the reverse date order, but I guess it makes sense for alphabetic order...

Reply:
That was my rationale, to distinguish older/newer bags, specially when recording and analysing. So are we Okay to leave it like that?

Reply:
Roger roger